apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,29 +625,31 @@ var ConversationView = {
     // to slide correctly. Bug 1009541
     this.cancelEdit();
 
-    if (Navigation.isCurrentPanel('thread')) {
```

Comment:
__note:__ there is a chance I'll return it or some-reincarnation-of-it back in https://github.com/mozilla-b2g/gaia/pull/30898/files#diff-33b9f27b0cd1246a89a6096fa1769c95R676 :(